manifests/testing/cdi-v1.2.0.yaml.in

```diff
@@ -0,0 +1,322 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: cdi-sa
+  namespace: {{ .Namespace }}
+  labels:
+    cdi.kubevirt.io: ""
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cdi
+  labels:
+    cdi.kubevirt.io: ""
+rules:
+- apiGroups: [""]
+  resources: ["events"]
+  verbs: ["create", "update", "patch"]
+- apiGroups: [""]
+  resources: ["persistentvolumeclaims"]
+  verbs: ["get", "list", "watch", "create", "update", "patch"]
+- apiGroups: [""]
+  resources: ["persistentvolumeclaims/finalizers"]
+  verbs: ["update"]
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list", "watch", "create", "delete"]
+- apiGroups: [""]
+  resources: ["pods/finalizers"]
+  verbs: ["update"]
+- apiGroups: [""]
+  resources: ["services"]
+  verbs: ["get", "list", "watch", "create", "delete"]
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "list", "watch", "create"]
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["get", "list", "watch", "create"]
+- apiGroups: ["cdi.kubevirt.io"]
+  resources:
+   - '*'
+  verbs:
+   - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: cdi-sa
+  labels:
+    cdi.kubevirt.io: ""
+roleRef:
+  kind: ClusterRole
+  name: cdi
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: cdi-sa
+    namespace: {{ .Namespace }}
+---
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: cdi-deployment
+  namespace: {{ .Namespace }}
+  labels:
+    cdi.kubevirt.io: ""
+    app: containerized-data-importer
+spec:
+  selector:
+    matchLabels:
+      app: containerized-data-importer
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: containerized-data-importer
+    spec:
+      serviceAccountName: cdi-sa
+      containers:
+      - name: cdi-controller
+        image: kubevirt/cdi-controller:v1.2.0
+        imagePullPolicy: IfNotPresent
+        args: ["-v=2"] # default verbosity; change to 2 or 3 for more detailed logging
```

Comment:
Do we want -v 2 in the default manifest? Was this debugging cruft?

Reply:
Since this manifest is just for running/testing locally, I thought `-v=2` would be good.  Can put back to 1 if you'd like though.